packages/vue-app/template/client.js

```diff
@@ -1,5 +1,5 @@
 import Vue from 'vue'
-import middleware from './middleware'
+import middleware from './middleware.js'
```

Comment:
@pi0 I was wondering too, but then checked other imports and they use .js. So it will make it more consistent with other imports. Or change other to be wihtout .js..

Reply:
@aldarund For valid esm outputs and consistently seems better idea having .js extension in the end of template imports 😊